README.md

```diff
@@ -57,102 +57,19 @@ _Preview means that this is a work in progress and NNCF does not guarantee the f
-
-# Instantiate your uncompressed model
-from tensorflow.keras.applications import ResNet50
-model = ResNet50()
-
-# Load a configuration file to specify compression
-nncf_config = NNCFConfig.from_json("resnet50_int8.json")
-
-# Provide dataset for compression algorithm initialization
-representative_dataset = tf.data.Dataset.list_files("/path/*.jpeg")
-nncf_config = register_default_init_args(nncf_config, representative_dataset, batch_size=1)
-
-# Apply the specified compression algorithms to the model
-compression_ctrl, compressed_model = create_compressed_model(model, nncf_config)
-
-# Now use compressed_model as a usual Keras model
-# to fine-tune compression parameters along with the model weights
-
-# ... the rest of the usual TensorFlow-powered training pipeline
-
-# Export to Frozen Graph, TensorFlow SavedModel or .h5  when done fine-tuning 
-compression_ctrl.export_model("compressed_model.pb", save_format='frozen_graph')
-```
-
-For a more detailed description of NNCF usage in your training code, see [this tutorial](docs/Usage.md). 
-For in-depth examples of NNCF integration, browse the [sample scripts](#compression-aware-training-samples) code, or the [example patches](#third-party-repository-integration) to third-party repositories.
-For FAQ, visit this [link](./docs/FAQ.md).
-
-### Usage examples of Post-Training Quantization
-
-NNCF provides [samples](#post-training-quantization-samples), which demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.
-
-To start the algorithm, provide the following entities:
-* Original model.
-* Validation part of the dataset.
-* [Data transformation function](./docs/compression_algorithms/post_training/Quantization.md#data-transformation-function) transforming data items from the original dataset to the model input data. 
-
-
-The basic workflow steps:
-1) Create the [data transformation function](./docs/compression_algorithms/post_training/Quantization.md#data-transformation-function).
+To run the algorithm you only need your model and the validation part of the dataset without labels. The basic workflow steps:
+1) Load a model and a data source.
 2) Create an instance of `nncf.Dataset` class by passing two parameters:
-* `data_source` - Iterable python object that contains data items for model calibration.
-* `transform_fn` - Data transformation function from the Step 1.
-3) Run the quantization pipeline.
+   * `data_source` — Iterable object containing data items for model calibration.
+   * `transform_fn` — [Data transformation function](./docs/compression_algorithms/post_training/Quantization.md#data-transformation-function), if your data source returns both images and labels.
+3) Run the quantization pipeline. It will return the quantized model for faster inference.
 
 Below are the usage examples for every backend.
 
-<details><summary><b>PyTorch</b></summary>
+<details open><summary><b>PyTorch</b></summary>
```

Comment:
I would recommend putting the OpenVINO backend in first place.

Reply:
I suppose PyTorch models are the most frequent target for NNCF PTQ among NNCF users. From that point of view PyTorch should be first.

Besides, this is way we'll attract users who mainly use PyTorch to try out OpenVINO since quantized models must be converted to OV IR to experience faster inference.

Reply:
IMHO: I don't agree with this statement, as far as I know we don't have statistics on PTQ API usage per backends yet. Anyway, OpenVINO PTQ is the base way to obtain a quantized model for OpenVINO because by using OpenVINO representation the user is guaranteed of getting an accurate model with best performance in OpenVINO than using the framework model representation. I think, we can firstly of all recommend what will give the user the best model for OpenVINO.

Reply:
I think it's safe to assume that the distribution of backends people would want to try NNCF with is strongly correlated with the popularity of the backends themselves. For OV and PT the number of downloads last month was 132K vs 10M. We should focus on a larger target audience.

I agree with your point, but as I see it, the purpose of the readme is to a greater degree to retain the visitors of the page and make them try out NNCF. For this, it is important to make the examples easy-to-follow and relatable. For most of the people PT is much more relatable then OV and they have their models in PT format and not OV format. I'm afraid a regular visitor wouldn't bother with the PT->OV conversion. 

I believe there's no information in the readme that the OV is the preferred backend. We should add such note so that the determined users would transition to OV.

Reply:
If using your logic TensorFlow should be first. Why is it not?

Reply:
While PT and TF downloads numbers are of the same order, NNCF experience with PT is better than with TF I believe.

Reply:
Placed OV first because PyTorch PTQ functionality is currently lagging behind the OV functionality